package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clever",
-  "version": "0.10.0",
+  "version": "0.11.0",
```

Comment:
It's backwards incompatible since it breaks the interface to the mocks in this package. Before the change, in order to use the mocks you needed to include understream as a peer dependency. Now you need to include highland.

I'm pretty sure semver treats all pre 1.0.0 minor releases as backwards incompatible, so this is safe.